packages/web/app/api/errors/route.ts

```diff
@@ -0,0 +1,29 @@
+import { Queue } from "bullmq";
+import { bull } from "../mq/bull";
+
+// do something with this later
+export const q = {
```

Comment:
well, we have a decision to be made. We need to keep track of the queues names. Where would this be the best place to be since we are consuming over the `ingest` package and over here?